content/configuration/node.md

```diff
@@ -5,9 +5,16 @@ contributors:
   - sokra
   - skipjack
   - oneforwonder
+  - Rob--W
 ---
 
-These options configure whether to polyfill or mock certain [Node.js globals](https://nodejs.org/docs/latest/api/globals.html) and modules. This allows code originally written for the Node.js environment to run in other environments like the browser. This feature is provided by webpack's internal [`NodeStuffPlugin`](https://github.com/webpack/webpack/blob/master/lib/NodeStuffPlugin.js).
+These options configure whether to polyfill or mock certain [Node.js globals](https://nodejs.org/docs/latest/api/globals.html) and modules.
+This allows code originally written for the Node.js environment to run in other environments like the browser.
+This feature is provided by webpack's internal
+[`NodeStuffPlugin`](https://github.com/webpack/webpack/blob/master/lib/NodeStuffPlugin.js) plugin.
+If the target is "web" (default) or "webworker", the
+[`NodeSourcePlugin`](https://github.com/webpack/webpack/blob/master/lib/node/NodeSourcePlugin.js)
+plugin is also activated.
```

Comment:
Done. Note that I did not just split the paragraph, I also added "plugin" after "NodeStuffPlugin", so that it has become "NodeStuffPlugin plugin".